src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,53 @@
 package core.basesyntax;
 
-public class WorkWithFile {
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
 
-    public void getStatistic(String fromFileName, String toFileName) {
+public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
 
+    public static void getStatistic(String fromFileName, String toFileName) {
+        int buySum = 0;
+        int supplySum = 0;
+        int result = 0;
+        File fileFrom = new File(fromFileName);
+        File fileTo = new File(toFileName);
+        StringBuilder stringBuilder = new StringBuilder();
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(fileFrom));
```

Comment:
go through common mistakes and check yourself carefully
you have mistakes from that list